modules/primer-navigation/lib/underline-nav.scss

```diff
@@ -0,0 +1,79 @@
+.UnderlineNav {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid $gray-200;
+}
+
+.UnderlineNav-items {
+  display: flex;
+  margin-bottom: -1px;
+}
+
+.UnderlineNav-item {
+  padding: $spacer-3 $spacer-2;
+  margin-right: $spacer-3;
+  font-size: $body-font-size;
+  line-height: $lh-default;
+  color: $text-gray;
+  text-align: center;
+  border-bottom: 2px solid transparent;
+
+  .Counter {
+    margin-left: $spacer-1;
+  }
+
+  .octicon {
```

Comment:
Similar thoughts here to nesting `Counter`. We could have an element class to account for it, such as `UnderlineNav-octicon` as utilities won't work because you need the hover styles.

Reply:
I removed the spacing from the component (same concern as Counter) but kept the color styles in under `.UnderlineNav-octicon`